examples/foreach-statement/foreach_statement.metatags

```diff
@@ -1,2 +1,2 @@
-description: This BBE demonstrates foreach statement in Ballerina.
-keywords: ballerina, ballerina by example, bbe, foreach, loops
+description: This BBE demonstrates the use of the `foreach` statement in Ballerina.
```

Comment:
@DamithaSenevirathne, this description should include the search criteria separated by commas.

Reply:
Updated in 7ff932554575d9d15c35bca3f2289cc5a9d2c133